mopro-ffi/Cargo.toml

```diff
@@ -17,13 +17,25 @@ default = []
 
 halo2 = []
 circom = [
-    "circom-prover",
-    "rust-witness",
+    "circom-wit-rustwitness",
+    "circom-proof-arkworks",
```

Comment:
I think it can just be `circom-rustwitness` and `circom-arkworks`
it looks too long

Reply:
My intention is to make the feature flag clear.

For developers who want to integrate with `mopro` in their own project, they can easily choose a witness lib and a proof generation lib without knowing which lib is for what. For example, if we use `circom-rustwitness` and `circom-arkworks`, it might happen that they choose `circom-arkworks` and `circom-rapidsnark` without any witness generation feature. Because `mopro-ffi` is the interface for developers, I prefer to make it clearer. What do you think?

Reply:
I think we just need to write down the documentation to make it clear
e.g. 
witness generator options: `witnesscalc`, `rustwitness`
proof generator options: `arkworks`, `rapidsnark`
I even think `witnesscalc`, `rustwitness`, `arkworks`, `rapidsnark` is enough
to not confused with `circom-prover` feature flags

and by default we can enable `rustwitness` and `arkworks` (more stable)
and only for advanced devs they can activate `witnesscalc` and `rapidsnark`
like
```toml
mopro-ffi = {version = "0.1", default-features = false, features = ["circom", "rapidsnark", "witnesscalc"]}
```
otherwise just use
```toml
mopro-ffi = {version = "0.1", features = ["circom"]}
```

Reply:
I'm not fan of reading docs and coding, but fixed as your suggestions in [7f96d08](https://github.com/zkmopro/mopro/pull/343/commits/7f96d0821834398b6086a9f9059ecfe0464f931d)